objectherkenning_openbare_ruimte/databricks_pipelines/common/utils.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+
+from databricks.sdk.runtime import *  # noqa: F403
+
+
+def delete_file(file_path):
+    """
+    Delete file from storage account.
+
+    Parameters:
+    file_path (str): The full path of the file using the volumes. The path should be within the /Volumes/ directory in Databricks.
+
+    """
+    try:
+        # list the file to check if it exists
+        dbutils.fs.ls(file_path)  # type: ignore[name-defined] # noqa: F821, F405
+        # if the file exists, remove it
+        dbutils.fs.rm(file_path)  # type: ignore[name-defined] # noqa: F821, F405
+        return True
+    except Exception as e:
+        # Check if the error is due to the file not being found
+        if "FileNotFoundException" in str(e):
+            print(f"File {file_path} does not exist, so it cannot be removed.")
+            return False
+        else:
+            # If there’s another type of error, raise it
+            raise RuntimeError(f"An unexpected error occurred: {e}")
+
+
+def get_image_name_from_detection_id(spark, catalog, schema, detection_id):
```

Comment:
This looks like a table function no? What is the logic to decide what goes in the table manager and what not?